src/main/java/games/strategy/engine/framework/map/download/DownloadFileProperties.java

```diff
@@ -59,7 +60,7 @@ private void setVersion(final Version v) {
   void setFrom(final DownloadFileDescription selected) {
     setVersion(selected.getVersion());
     props.setProperty("map.url", selected.getUrl());
-    props.setProperty("download.time", new Date().toString());
+    props.setProperty("download.time", TimeManager.toDateString(LocalDateTime.now()));
```

Comment:
This produces slightly different results on my system:

```
Old: Mon Jun 19 01:23:50 EDT 2017
New: Mon Jun 19 01:23:50 ET 2017
```

It doesn't appear that the `download.time` property is ever used (parsed) by the code.  Can you please confirm?